model/src/lib.rs

```diff
@@ -29,13 +24,27 @@ use std::ops::Sub;
 use std::str;
 use std::time::SystemTime;
 use std::time::UNIX_EPOCH;
-use time::OffsetDateTime;
-use time::PrimitiveDateTime;
-use time::Time;
 
-use self::cfd::Role;
+#[macro_use]
+mod sqlx_ext;
 
 pub mod cfd;
+pub mod olivia;
+pub mod payout_curve;
+
+/// The interval until the cfd gets settled, i.e. the attestation happens
+///
+/// This variable defines at what point in time the oracle event id will be chose to settle the cfd.
+/// Hence, this constant defines how long a cfd is open (until it gets either settled or rolled
+/// over).
+///
+/// Multiple code parts align on this constant:
+/// - How the oracle event id is chosen when creating an order (maker)
+/// - The sliding window of cached oracle announcements (maker, taker)
+/// - The auto-rollover time-window (taker)
+pub const SETTLEMENT_INTERVAL: time::Duration = time::Duration::hours(24);
+
+pub const N_PAYOUTS: usize = 200;
```

Comment:
This is an application decision IMO and should maybe stay in `daemon`?

Reply:
Yeah, you're right. I'll return it to `daemon` and add a similar constant in the `model` tests.